oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java

```diff
@@ -229,6 +243,9 @@ public StorageToMeasure(MetadataRegistry.Schema schema, DataPoint dataPoint) {
 
         @Override
         public Object get(String fieldName) {
+            if (fieldName.equals(Metrics.TIME_BUCKET)) {
```

Comment:
`Record` and `Metrics` seem to have duplicated **TIME_BUCKET** fields. I think we need a PR to merge these two into `StorageData`, and use it instead of here.

Reply:
Yes. I think it should be moved to the base class. We may do this later? We have a lot of references to these constants

Reply:
Yes, that should be on a separate PR, no matter before or after.

Reply:
Done through https://github.com/apache/skywalking/pull/10208